quicktests/overlaying/tests/realistic/numeric_grid.js

```diff
@@ -26,8 +26,21 @@ function run(svg, data, Plottable) {
       .x2(function(d){ return d.x2; }, xScale)
       .y2(function(d) { return d.y2; }, yScale)
       .attr("fill", function(d) { return d.fill; })
+      .attr("opacity", .5)
       .attr("stroke", function(){ return "#000000"; })
       .attr("stroke-width", function(){ return 4; });
 
   plot.renderTo(svg);
+
+  var hover = new Plottable.Interactions.Pointer();
+  hover.onPointerMove(function(p){
+    plot.entities().forEach(function(entity) {
+      entity.selection.attr("opacity", .3);
+    });
+    var entities = plot.entitiesAt(p);
+    if (entities.length > 0) {
+      entities[0].selection.attr("opacity", 1);
+    }
+  })
```

Comment:
This could be done a bit more efficiently by using `onPointerEnter`, `onPointerExit`, and cacheing the last set of `Entities`:
```Javascript
hover.onPointerEnter(function() {
  plot.entities().forEach(function(entity) { entity.selection.attr("opacity", 0.3); });
});
hover.onPointerMove(function(p) {
  lastEntities.forEach(function(entity) { entity.selection.attr("opacity", 0.3); });
  var entities = plot.entitiesAt(p);
  entities.forEach(function(entity) { entity.selection.attr("opacity", 1); });
  lastEntities = entities;
});
hover.onPointerExit(function() {
  plot.entities().forEach(function(entity) { entity.selection.attr("opacity", 1); });
  lastEntities = [];
});
```